cromshell

```diff
@@ -38,13 +38,13 @@ TASK_COLOR_FAILED='\033[0;31m'
 CROMSHELL_CONFIG_DIR=${HOME}/.cromshell
 mkdir -p ${CROMSHELL_CONFIG_DIR}
 CROMWELL_SUBMISSIONS_FILE="${CROMSHELL_CONFIG_DIR}/all.workflow.database.tsv"
-[ ! -f ${CROMWELL_SUBMISSIONS_FILE} ] && echo -e "DATE\tCROMWELL_SERVER\tRUN_ID\tWDL_NAME\tSTATUS" > ${CROMWELL_SUBMISSIONS_FILE} 
+[[ ! -f ${CROMWELL_SUBMISSIONS_FILE} ]] && echo -e "DATE\tCROMWELL_SERVER\tRUN_ID\tWDL_NAME\tSTATUS" > ${CROMWELL_SUBMISSIONS_FILE}
 
 # Find the CROMWELL Server:
 CROMWELL_SERVER_FILE=${CROMSHELL_CONFIG_DIR}/cromwell_server.config
 CROMWELL_NEEDS_SETUP=true
-[ -f ${CROMWELL_SERVER_FILE} ] && CROMWELL_NEEDS_SETUP=false
-if ! $CROMWELL_NEEDS_SETUP ; then
+[[ -f ${CROMWELL_SERVER_FILE} ]] && CROMWELL_NEEDS_SETUP=false
+if ! ${CROMWELL_NEEDS_SETUP} ; then
```

Comment:
Same convention question here:

What conventions are you using / do you want to enforce for variables?

When do we want to mandate curly braces as opposed to using direct variable specification (i.e. `${VAR}` vs `$VAR`?